src/comp/comp.py

```diff
@@ -24,48 +24,52 @@ def __repr__(self):
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [i.name for i in humans if i.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [i.name for i in humans if i.name[-1:] == 'e']
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+cglist = ['C', 'D', 'E', 'F', 'G']
+c = [i.name for i in humans if i.name[0] in cglist]
 print(c)
 
 # Write a list comprehension that creates a list of all the ages plus 10.
 print("Ages plus 10:")
-d = []
+d = [i.age + 10 for i in humans]
 print(d)
 
 # Write a list comprehension that creates a list of strings which are the name
 # joined to the age with a hyphen, for example "David-31", for all humans.
 print("Name hyphen age:")
-e = []
+e = [i.name + '-' + str(i.age) for i in humans]
 print(e)
 
 # Write a list comprehension that creates a list of tuples containing name and
 # age, for example ("David", 31), for everyone between the ages of 27 and 32,
 # inclusive.
 print("Names and ages between 27 and 32:")
-f = []
+#def convert(f):       #THANKS GOOGLE..not
+   # return tuple(f) 
+f = [(i.name, i.age) for i in humans if i.age >= 27 and i.age <= 32]
```

Comment:
Great job in general with these list comprehensions.  They can be quite powerful.

Maybe change "i" to "human" to be more readable? ` (human.name, human.age) for human in humans` .. but really that's just preference, you did a great job and i'm having trouble finding anything else to comment on for things to improve on.  LOL good work.